Common/include/CConfig.hpp

```diff
@@ -805,8 +805,9 @@ class CConfig {
   Gas_Constant,         /*!< \brief Specific gas constant. */
   Gas_ConstantND,       /*!< \brief Non-dimensional specific gas constant. */
   *Molecular_Weight;    /*!< \brief Molecular weight of an incompressible ideal gas (g/mol). */
-  unsigned short nMolecular_Weight; /*!< \brief Number of species molecular weights. */
-  su2double Specific_Heat_Cp, /*!< \brief Specific heat at constant pressure. */
+  unsigned short nMolecular_Weight, /*!< \brief Number of species molecular weights. */
+  nSpecific_Heat_Cp;              /*!< \brief Number of species specific heat constants at constant pressure. */
+  su2double *Specific_Heat_Cp, /*!< \brief Specific heat at constant pressure. */
   Specific_Heat_CpND,         /*!< \brief Non-dimensional specific heat at constant pressure. */
   Specific_Heat_Cv,           /*!< \brief Specific heat at constant volume. */
   Specific_Heat_CvND,         /*!< \brief Non-dimensional specific heat at constant volume. */
```

Comment:
Don't you have to do something for Cv?
It looks like it is not used anywhere in the code, @bigfooted should we remove it to avoid future problems?

Reply:
Hi Pedro, I commented this to Nijso that they are not used in the code. we were thinking to remove them in a future pull request.

Reply:
Yes please remove it, it is not used anywhere and it's related to this PR since after this it really can't be used.

Reply:
got it, thank you so much for the comments.